src/mixins/PreviewUrl.js

```diff
@@ -75,6 +75,10 @@ export default {
 				if (isPublic()) {
 					return generateUrl(`/apps/files_sharing/publicpreview/${getToken()}?file=${encodeFilePath(filename)}&${searchParams}`)
 				}
+				if (filename.startsWith('/photos/')) {
```

Comment:
Filename is the full dav path btw, including the collection. So for a normal file it looks like `/files/user/...` and `/photos/user/...` for collaborative albums.